gradle/dependencies.gradle

```diff
@@ -39,7 +39,7 @@ versions += [
   rocksDB: "4.1.0",
   scalaTest: "2.2.6",
   scalaParserCombinators: "1.0.4",
-  slf4j: "1.7.15",
+  slf4j: "1.7.16",
```

Comment:
@ijuma interesting...version 1.7.17 was skipped and they are releasing really frequently. Its almost as if they release as soon as any single issue is fixed. 

1.7.18 includes a thread safety fix: http://jira.qos.ch/browse/SLF4J-356

I can bump the version here. Guess we should just bump right before release given the frequent updates.